money-monkey-web/lib/money.ts

```
@@ -3,7 +3,10 @@ export const usdFormatter = new Intl.NumberFormat("en-US", {
   currency: "USD",
 });
 
-export function formatMoney(amountCents: number): string {
+export function formatMoney(amountCents: number | string): string {
+  if (typeof amountCents === "string") {
+    amountCents = Number.parseInt(amountCents);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)